microsoft/testsuites/storage/performance.py

```diff
@@ -0,0 +1,189 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+from enum import Enum
+from typing import Any, Dict, List
+
+from assertpy import assert_that
+
+from lisa import (
+    Logger,
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    notifier,
+    schema,
+    search_space,
+    simple_requirement,
+)
+from lisa.environment import Environment
+from lisa.features import Disk
+from lisa.notifier import FIOResult, FIOResultsMessage
+from lisa.tools import Fdisk, Fio, Lscpu, Mdadm
+
+FIOMODES = Enum(
```

Comment:
If it's moved to fio, it's easy to share.

Reply:
updated